noxfile.py

```diff
@@ -64,10 +90,24 @@ def blacken(session):
     session.run("black", *BLACK_PATHS)
 
 
-@nox.session(python=["2.7", "3.5", "3.6", "3.7"])
+@nox.session(python=["3.5", "3.6", "3.7"])
 def unit(session):
     session.install(*TEST_DEPENDENCIES)
     session.install(".")
+    session.run(
+        "pytest",
+        "--cov=google.auth",
+        "--cov=google.oauth2",
+        "--cov=tests",
+        "tests",
+        "tests_async",
+    )
+
+
+@nox.session(python="2.7")
+def unit2(session):
```

Comment:
maybe name `unit-2-7` to better match the names made for the other unit test run?

Reply:
Resolved to include both python 3.5 and 2.7 and changed name accordingly